### PR TITLE
Express Updates are Dead

### PR DESCRIPTION
**Pull request by bryandam**
Had this come up on reddit and although everyone kind of knows that Express Updates are dead ... there's frightfully little I can find in any official documentation saying so.  The best anyone could come up with is this article stating that 'the next release' (1809) won't have them.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@bryandam : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@mestew Could you take a look at a proposed addition to your article? Thanks.

**Comment by ktoliver** [comment]
@mestew for review

Thanks.

File: memdocs/configmgr/sum/deploy-use/manage-express-installation-files-for-windows-10-updates.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ ms.assetid: b8d8af88-e8ac-4deb-921b-975e2d2afd80
 
 
 ---
+> [!NOTE]    
+> The release of Windows 10 1809 utilizes a new update delivery method that eliminated Express Updates.  To learn more, see (What’s next for Windows 10 and Windows Server quality updates)[https://techcommunity.microsoft.com/t5/windows-it-pro-blog/what-s-next-for-windows-10-and-windows-server-quality-updates/bc-p/234768].
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/sum/deploy-use/manage-express-installation-files-for-windows-10-updates.md:17]:
```suggestion
> The release of Windows 10 Version 1809 utilizes a new update delivery method that eliminated Express Updates.  To learn more, see [What’s next for Windows 10 and Windows Server quality updates](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/what-s-next-for-windows-10-and-windows-server-quality-updates/bc-p/234768).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - this PR is quite old. I am not sure if it is still relevant.

**Comment by bryandam** [comment]
@JohanFreelancer9: I think the PR is still relevant though I'd argue it doesn't go far enough.

The problem is that the ConfigMgr product still has a feature to handle Express Updates and the docs still tell you how to use and why you'd want to use it without telling you that the OS Feature itself is long dead.

Honestly, my initial thought was just to replace the whole page with a big 72-point warning "It's Dead Jim" but I know Microsoft has this thing about 'professionalism' and all that.

**Comment by USCloud-EricWilborn** [comment]
> @JohanFreelancer9: I think the PR is still relevant though I'd argue it doesn't go far enough.
> 
> The problem is that the ConfigMgr product still has a feature to handle Express Updates and the docs still tell you how to use and why you'd want to use it without telling you that the OS Feature itself is long dead.
> 
> Honestly, my initial thought was just to replace the whole page with a big 72-point warning "It's Dead Jim" but I know Microsoft has this thing about 'professionalism' and all that.
> 
> At the end of the day, what we need, it something we can point ConfigMgr users to that says this feature should no longer be used. It needs to be 'docs official' rather than institutional knowledge. How we achieve that I leave in your capable hands.

Has there been any update on this request?

**Comment by bryandam** [comment]
<looks at live doc>
I'd say ... no. There's been no update.

I don't have a  ConfigMgr console in front of me at the moment but if the UI still exists then the docs should make it clear that the feature itself is dead and no good can come of enabling it.
If the UI is removed then the whole doc should just die and go away.

**Comment by JohanFreelancer9** [comment]
@yogkumgit, could you please follow up? Many thanks.

**Comment by yogkumgit** [comment]
@mestew Please comment and help. Thanks

**Pull request closed without merging**